### PR TITLE
Updating description for the Descheduler Operator

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ingvagabund Here are updates for the operator description if you can please take a look. Let me know if there's anything else I need to do for this, or if you have any questions on the updates. Thanks!

File: manifests/4.4/cluster-kube-descheduler-operator.v4.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -39,100 +39,34 @@ spec:
...
-      | `lownodeutilization` | `LowNodeUtilization` |
-      | `nodeaffinity` | `RemovePodsViolatingNodeAffinity` |
-      | `nodetaints` | `RemovePodsViolatingNodeTaints` |
+      * `LowNodeUtilization` - This strategy finds nodes that are underutilized and evicts pods, if possible, from other nodes in the hope that recreation of evicted pods will be scheduled on these underutilized nodes.
+      * `RemoveDuplicates` - This strategy ensures that there is only one pod associated with a ReplicaSet, ReplicationController, Deployment, or Job running on same node. If there are more, then those duplicate pods are evicted for better spreading of pods in a cluster.
+      * `RemovePodsViolatingInterPodAntiAffinity` - This strategy ensures that pods violating inter-pod anti-affinity are removed from nodes.
+      * `RemovePodsViolatingNodeAffinity` - This strategy ensures that pods violating node affinity are removed from nodes.
+      * `RemovePodsViolatingNodeTaints` - This strategy ensures that pods violating `NoSchedule` taints on nodes are removed.
 
-      ## Sample CR
```
Comment (ingvagabund) [inline manifests/4.4/cluster-kube-descheduler-operator.v4.4.0.clusterserviceversion.yaml]: Just one nit. Let's put `These strategies are documented in detail in the [descheduler README](https://github.com/kubernetes-sigs/descheduler/#policy-and-strategies).` right at the end of `## Descheduler Strategies` section.

**Comment by ingvagabund** [comment]
Merged as part of https://github.com/openshift/cluster-kube-descheduler-operator/pull/94
/close
@bergerhoffer thanks for the changes!!!

**Pull request closed without merging**